tests/chart_tests/test_elasticsearch.py

```diff
@@ -205,3 +205,34 @@ def test_nginx_es_client_network_selector_with_logging_sidecar_enabled(
                 },
             },
         ] == doc["spec"]["ingress"][0]["from"]
+
+    def test_nginx_es_index_pattern_with_sidecar_logging_enabled(self, kube_version):
+        """Test Nginx ES Service Index Pattern Search with sidecar logging."""
+        docs = render_chart(
+            kube_version=kube_version,
+            values={"global": {"loggingSidecar": {"enabled": True}}},
+            show_only=[
+                "charts/elasticsearch/templates/nginx/nginx-es-configmap.yaml",
+            ],
+        )
+
+        assert len(docs) == 1
+        doc = docs[0]
+        es_index = doc["data"]["nginx.conf"]
+        assert doc["kind"] == "ConfigMap"
+        assert "vector.$remote_user.*/$1" in es_index
+
+    def test_elasticsearch_exporter_securitycontext_defaults(self, kube_version):
```

Comment:
Can we get one more test here that uses a custom `.Values.securityContext.runAsUser`?

Reply:
yes sure